cypress/e2e/cypress/default/debug-tab/checkpoint-value.cy.ts

```diff
@@ -2,7 +2,7 @@ describe('Stream is not closed prematurely', () => {
   // Make this beforeEach() when issue https://github.com/wearefrank/ladybug/issues/344
   // will have been fixed.
   before(() => {
-    cy.apiDeleteAll('FileDebugStorage')
+    cy.apiDeleteAll(Cypress.env('debugStorageName') as string)
```

Comment:
Cast je dit omdat het undefined kan zijn? Zo ja, is het in dat geval niet beter om er een if statement omheen te zetten?

Reply:
Deze waarde wordt doorgegeven vanuit GitHub Actions. We weten dus zeker dat het een string is, maar de functiie Cypress.env() heeft als return type any. Je zou dit kunnen wrappen met een custom command zodat deze cast maar op één plek staat, maar dan zie je niet meer meteen dat de Cypress environment wordt bevraagd. Zoals het nu is is het dus het minste kwaad. Dit is onze gezamenlijke conclusie na mondeling overleg.